packages/editor-service/package.json

```diff
@@ -17,8 +17,9 @@
     "build": "tsc",
     "prepublishOnly": "npm run clean && npm run build",
     "lint": "tslint --project ./tsconfig.json",
-    "test": "npm run clean && npm run build && mocha && npm run lint",
-    "test:watch": "tsc-then -- mocha",
+    "test": "npm run clean && npm run build && npm run test:unit && npm run lint",
+    "test:unit": "mocha",
+    "test:watch": "tsc-then -- npm run test:unit",
```

Comment:
running `mocha` directly is generally better in test:watch, because if there's a failure when doing `npm run foo` then npm outputs like 20-30 lines about how the failure isn't npm's fault

Reply:
ok.